db/migrate/20210616102540_create_bursaries.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateBursaries < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bursaries do |t|
+      t.string :training_route, null: false
+      t.string :subject, null: false
```

Comment:
Updated based on our offline discussions. There is now a join table `bursary_subjects`. It ended up making most sense to keep the `amount` on the `bursary` since this is essentially what a bursary is! An amount of money, linked to a subject.

It also meant I could do `allocation_subject.bursaries.find_by(training_route: route).amount`, linking through the `bursary_subjects`, which i think reads nicely.